aws/data_source_aws_eip_test.go

```diff
@@ -152,6 +152,26 @@ func TestAccDataSourceAwsEip_Instance(t *testing.T) {
 	})
 }
 
+func TestAccDataSourceAWSEIP_CarrierIP(t *testing.T) {
```

Comment:
Looks like the capitalization of this new test and the one below differs from the rest of the file. Been thinking of making a linter for these inconsistencies anyways.

Reply:
There's a general issue for naming conventions: https://github.com/hashicorp/terraform-provider-aws/issues/13128.